mailbox/api/src/test/java/org/apache/james/mailbox/ApplicableFlagBuilderTest.java

```diff
@@ -22,25 +22,25 @@
 
 import javax.mail.Flags;
 
-import org.assertj.core.api.JUnitSoftAssertions;
-import org.junit.Rule;
-import org.junit.Test;
+import org.assertj.core.api.JUnitJupiterSoftAssertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
 
 import com.google.common.collect.ImmutableList;
 
-public class ApplicableFlagBuilderTest {
+class ApplicableFlagBuilderTest {
 
-    @Rule
-    public final JUnitSoftAssertions softly = new JUnitSoftAssertions();
+    @RegisterExtension
+    public final JUnitJupiterSoftAssertions softly = new JUnitJupiterSoftAssertions();
```

Comment:
It worth rewriting them using the "new" way to do soft assertions

Reply:
That is ?

Reply:
Like this you mean?
```
SoftAssertions.assertSoftly(softly -> {
// soft asserts...
});
```

Alright!

Reply:
Remembered doing the same in an other similar refactoring.... Just pushed an extra commit to fix it as well, thanks for the feedback